### PR TITLE
Add note on rounding mode when optimizing an average using the database

### PR DESCRIPTION
**Pull request by basdebakker**

### CODE REVIEW

File: content/howto/logic-business-rules/optimizing-microflow-aggregates.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ The Mendix Platform only creates an optimized SQL query if the list is not used
 
 If you do want to use the list but you also want the optimized query, do two separate retrieves. This will do the optimized query, and you can use the second retrieve in your microflow.
 
+When an average is calculated in this way by using an optimized SQL query, the rounding mode configured in the project settings will not be respected. The result will be rounded by the database in a database specific manner. If the rounding mode is essential for the result, you can retrieve the sum and count separately and perform the division in the microflow.
```
Comment (snajahi) [inline content/howto/logic-business-rules/optimizing-microflow-aggregates.md]: Perhaps the first part of the sentence could be reworded like this _When an average is calculated within an optimised SQL query_.

**Review by snajahi — APPROVED** [review]

**Review by snajahi — APPROVED** [review]

**Review by snajahi — APPROVED** [review]

**Review by snajahi — APPROVED** [review]

File: content/howto/logic-business-rules/optimizing-microflow-aggregates.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ The Mendix Platform only creates an optimized SQL query if the list is not used
 
 If you do want to use the list but you also want the optimized query, do two separate retrieves. This will do the optimized query, and you can use the second retrieve in your microflow.
 
+When an average is calculated using an optimized SQL query, the rounding mode configured in the project settings will not be respected. The result will be rounded by the database in a database specific manner. If the rounding mode is essential for the result, you can retrieve the sum and count separately and perform the division in the microflow.
```
Comment (ilagordhan) [inline content/howto/logic-business-rules/optimizing-microflow-aggregates.md:19]:
```suggestion
When an average is calculated within an optimized SQL query, the rounding mode configured in the project settings will not be respected. The result will be rounded by the database according to the database settings. If the rounding mode is essential for the result, you can retrieve the sum and count separately and perform the division in the microflow.
```

**Review by ilagordhan — APPROVED** [review]

File: content/howto/logic-business-rules/optimizing-microflow-aggregates.md (hunk 1 above)
Comment (ilagordhan) [reply content/howto/logic-business-rules/optimizing-microflow-aggregates.md]: I have incorporated you review comment and committed a suggested change.

File: content/howto/logic-business-rules/optimizing-microflow-aggregates.md (hunk 2 above)
Comment (basdebakker) [reply content/howto/logic-business-rules/optimizing-microflow-aggregates.md:19]: It is the optimized query as a whole that returns the average, so I'm not keen on the "within".

File: content/howto/logic-business-rules/optimizing-microflow-aggregates.md [hunk 3]
```diff
@@ -16,6 +16,8 @@ The Mendix Platform only creates an optimized SQL query if the list is not used
 
 If you do want to use the list but you also want the optimized query, do two separate retrieves. This will do the optimized query, and you can use the second retrieve in your microflow.
 
+When an average is calculated within an optimized SQL query, the rounding mode configured in the project settings will not be respected. The result that is returned will be rounded by the database according to the database settings. If the rounding mode is essential for the result, you can retrieve the sum and count separately and perform the division in the microflow.
```
Comment (ilagordhan) [inline content/howto/logic-business-rules/optimizing-microflow-aggregates.md:19]:
```suggestion
When an average is calculated within an optimized SQL query, the rounding mode configured in the project settings will not be respected. The result that is returned will be rounded by the database according to the database settings. If the rounding mode setting for the project is essential for the result, you can retrieve the sum and count separately and perform the division in the microflow.
```

**Review by ilagordhan — APPROVED** [review]

**Pull request merged**